cypress/Pages/BackHomePage.js

```diff
@@ -0,0 +1,14 @@
+export class BackHomePage {
+
+    BackHomeButton = '#back-to-products'
+    ProductLogo = '.title'
```

Comment:
productLogo

Reply:
I have changed the all variables name conventions as per suggestion and code standard